src/components/Sidebar/SidebarContext.tsx

```diff
@@ -0,0 +1,66 @@
+import { createContext, ReactNode, useContext, useMemo, useState } from 'react';
+
+import { MapKey } from './Sidebar';
+
+interface SelectedMapTypeState {
+  selectedMapType: MapKey;
+  setSelectedMapType: (value: MapKey) => void;
+}
+
+interface SelectedAlertsState {
+  selectedAlert: string;
+  setSelectedAlert: (value: string) => void;
+  isAlertSelected: (alertType: string) => boolean;
+  toggleAlert: (alertType: string) => void;
+}
+
+interface SidebarState {
+  isSidebarOpen: boolean;
+  toggleSidebar: () => void;
+  openSidebar: () => void;
+  closeSidebar: () => void;
+}
```

Comment:
interfaces go to `domain/entries/Sidebar`

Reply:
I moved SidebarContext.tsx to `domain/contexts/SidebarContext.tsx`. I left the intefaces in the file like it is in `CountryContext.tsx`